### PR TITLE
Starred button

### PR DESCRIPTION
**Pull request by ianb**
Also onboarding flow, and a small tweak to bin/release-version

### CODE REVIEW

File: addon/webextension/background/main.js [hunk 1]
```diff
@@ -14,6 +14,9 @@ window.main = (function () {
 
   browser.storage.local.get(["hasSeenOnboarding"]).then((result) => {
     hasSeenOnboarding = !! result.hasSeenOnboarding;
+    if (! hasSeenOnboarding) {
+      setIconActive(false, null);
```
Comment (dannycoates) [inline addon/webextension/background/main.js]: nit: maybe we could leave the `null` off of these calls and default `tabId` to null. later is fine

File: addon/webextension/background/main.js [hunk 2]
```diff
@@ -58,6 +68,11 @@ window.main = (function () {
 
   browser.browserAction.onClicked.addListener(catcher.watchFunction((tab) => {
     if (shouldOpenMyShots(tab.url)) {
+      if (! hasSeenOnboarding) {
+        sendEvent("goto-onboarding", "selection-button");
```
Comment (dannycoates) [inline addon/webextension/background/main.js]: do we need to check the telemetry pref first here?

File: addon/webextension/background/main.js [hunk 3]
```diff
@@ -35,7 +42,10 @@ window.main = (function () {
   }
 
   function setIconActive(active, tabId) {
-    const path = active ? "icons/icon-highlight-38.png" : "icons/icon-38.png";
+    let path = active ? "icons/icon-highlight-38.png" : "icons/icon-38.png";
+    if ((! hasSeenOnboarding) && ! active) {
+      path = "icons/icon-38-starred.png";
```
Comment (pdehaan) [inline addon/webextension/background/main.js]: nit: This naming feels off to me. icon-highlight-38.png vs icon-38-starred.png

File: addon/webextension/background/main.js (hunk 1 above)
Comment (ianb) [inline addon/webextension/background/main.js]: Technically we could leave it off and it'll default to undefined right now!  But I don't know, how can you know what's right in JavaScript, amiright?

**Pull request merged**